content/en/agent/troubleshooting/agent_check_status.md

```diff
@@ -84,16 +92,10 @@ Run the following script, with the proper `<CHECK_NAME>`:
 
 For example, to run the disk check:
 
-For Agent versions <= 6.11:
 ```
 C:\Program' 'Files\Datadog\Datadog' 'Agent\embedded\python.exe C:\Program' 'Files\Datadog\Datadog' 'Agent\agent\agent.py check disk
 ```
 
-For Agent versions <= 6.12:
-```
-C:\Program' 'Files\Datadog\Datadog' 'Agent\embedded2\python.exe C:\Program' 'Files\Datadog\Datadog' 'Agent\agent\agent.py check disk
-```
```

Comment:
this didn't make sense because it was in a "v>=5.12" tab (only applied to Agent v5)